debian_package/sbin/folder2ram

```diff
@@ -234,67 +235,40 @@ exit
 EOF
 
   #chmodding this script to make it only root/group-writable and root/group-executable
-  chmod 774 "/etc/init.d/folder2ram_temporary"
+  chmod 755 "/etc/init.d/folder2ram_temporary"
 
 } #### END write_cleanup_initscript
 
 write_systemd_startup_service() {
 
-  if [ "$timeout_setting" != '' ]; then
-    timeout_line="TimeoutSec=$timeout_setting"
-  fi
+  #read timeout from configuration file
+  local timeOUT=$(read_config_file | grep "^TIMEOUT" | awk -F'=' '{print $2}')
+  [ "${timeOUT}" != "" ] || timeOUT=120
 
   #writing a tiny amount of stuff to have systemd do the same thing as above.
   #clear symptom that systemd is master race.
   cat <<EOF >"$systemd_startup_service_file"
 [Unit]
 Description=folder2ram systemd service
 After=local-fs.target
-After=blk-availability.service
-DefaultDependencies=no
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/sbin/folder2ram -mountall
-$timeout_line
+ExecStart=$(which journalctl) --relinquish-var
+ExecStart=-${binFILE} -mountall
+ExecStart=$(which journalctl) --flush
+ExecStop=$(which journalctl) --relinquish-var
+ExecStop=-${binFILE} -umountall
+ExecStop=$(which journalctl) --flush
+TimeoutSec=${timeOUT}
 
 [Install]
 WantedBy=basic.target
 EOF
 
   #chmodding this file to make it only root/group-writable
-  chmod 664 "$systemd_startup_service_file"
-
-} #### END write_systemd_service
-
-write_systemd_shutdown_service() {
-
-  if [ "$timeout_setting" != '' ]; then
-    timeout_line="TimeoutSec=$timeout_setting"
-  fi
-
-  #writing a tiny amount of stuff to have systemd do the same thing as above.
-  #clear symptom that systemd is master race.
-
-  cat <<EOF >"$systemd_shutdown_service_file"
-[Unit]
-Description=folder2ram systemd service
-After=blk-availability.service
-BindsTo=blk-availability.service
-
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-ExecStop=/sbin/folder2ram -umountall
-$timeout_line
-
-[Install]
-WantedBy=multi-user.target
-EOF
-
-  #chmodding this file to make it only root/group-writable
-  chmod 664 "$systemd_shutdown_service_file"
```

Comment:
It's been a while so I don't remember the details anymore. But I remember that I originally had a single systemd unit to start/stop the script but I had to break them up in two because I saw that they needed different After= BindsTo=  to be run at the right time.

Quite frankly, I don't want to invest the time to throughly re-test this change on a OpenMediaVault setup like mentioned in the issues in the plugin I mentioned in another comment. I'm sorry but this script is used by a sizeable amount of people and I don't want to break stuff or invest a lot of testing time just for the sake of refactoring things that work fine as-is.

So please remove this change, leave the two systemd units one for start and one for stop